src/playdate/scoreboards.nim

```diff
@@ -56,8 +61,7 @@ template invokeCallback(callbackSeqs, value, errorMessage, freeValue, builder: u
     type ResultType = typeof(builder)
     let callback = callbackSeqs.pop()
     if value == nil:
-        let message = if errorMessage == nil: "Playdate-nim: No value provided for callback" else: $errorMessage
-        callback(PDResult[ResultType](kind: PDResultError, message: message))
+        callback(PDResult[ResultType](kind: PDResultUnavailable))
```

Comment:
It looks like we will never send back a `PDResultError` with this change -- I think we still need to check if `errorMessage` is nil here.

Reply:
Yikes, thanks for catching this